ui/packages/consul-ui/app/components/hashicorp-consul/index.js

```diff
@@ -8,4 +8,10 @@ import { inject as service } from '@ember/service';
 
 export default class HashiCorpConsul extends Component {
   @service('flashMessages') flashMessages;
+  @service('env') env;
+
+  get consulVersion() {
+    const suffix = this.env.var('CONSUL_BINARY_TYPE') !== 'oss' && this.env.var('CONSUL_BINARY_TYPE') !== '' ? '+ent' : '';
```

Comment:
This approach will only ever show the `+ent` suffix, but we could just pass through the binary type string here and let it display as the version suffix no matter what, which could be useful for displaying things like `+dev` or other binary targets.